posthog/models/app_metrics2/sql.py

```diff
@@ -0,0 +1,221 @@
+from django.conf import settings
```

Comment:
Note that everything here was copied from `app_metrics` (1), the name, schema and queries were adjusted as-is. Queries aren't used yet, porting over will happen later.